app/apphandlers/embeddedAppHandler.go

```diff
@@ -253,8 +252,10 @@ func (a *embeddedApp) StartRudderCore(ctx context.Context, options *app.Options)
 		MultiTenantStat: multitenantStats,
 	}
 
-	rateLimiter := ratelimiter.HandleT{}
-	rateLimiter.SetUp()
+	rateLimiter, err := gwThrottler.New(stats.Default)
+	if err != nil {
+		return fmt.Errorf("failed to create gw rate limiter: %w", err)
```

Comment:
I like that in case of an error we allow the request to pass, but we might need a metric or setup an alert based on that log.

Reply:
Here we aren't passing the requests right? We just return from the app handler and the server would restart. If you meant passing the request in gateway component yes we can set up an alert on the log/add a metric